mbuild/compound.py

```diff
@@ -43,8 +43,8 @@ def load(filename, relative_to_module=None, compound=None, coords_only=False,
 
     Parameters
     ----------
-    filename : str
-        Name of the file from which to load atom and bond information.
+    filename_or_object : str
```

Comment:
We should change what the type expected is:
```suggestion
    filename_or_object : str, mdtraj.Trajectory, parmed.Structure
```